test/quark-core/Noncer.t.sol

```diff
@@ -92,7 +92,7 @@ contract NoncerTest is Test {
         bytes memory result = aliceWallet.executeQuarkOperation(op, v, r, s);
 
         (bytes32 submissionTokenResult) = abi.decode(result, (bytes32));
-        assertEq(submissionTokenResult, EXHAUSTED_TOKEN);
+        assertEq(submissionTokenResult, op.nonce);
```

Comment:
Consider also asserting that the actual value is set to `EXHAUSTED_TOKEN` in the `NonceManager`.

Reply:
added